src/gui/optionsdlg.ui

```diff
@@ -2362,10 +2362,124 @@
                 </spacer>
                </item>
                <item row="3" column="0" colspan="3">
-                <widget class="QCheckBox" name="checkIgnoreSlowTorrentsForQueueing">
-                 <property name="text">
+                <widget class="QGroupBox" name="checkIgnoreSlowTorrentsForQueueing">
+                 <property name="title">
                   <string>Do not count slow torrents in these limits</string>
                  </property>
+                 <property name="checkable">
+                  <bool>true</bool>
+                 </property>
+                 <property name="checked">
+                  <bool>false</bool>
+                 </property>
+                 <layout class="QGridLayout" name="gridLayout_8">
```

Comment:
I'm getting this error:
>optionsdlg.ui: Warning: The name 'gridLayout_8' (QGridLayout) is already in use, defaulting to 'gridLayout_81'.

Can you fix it?

Reply:
Yes